### DISCUSSION TITLE
How to know the size of items that were bookmarked.

### DISCUSSION THREAD

**Discussion by danpos** [sepinf-inc/IPED#183] [Q&A] [answered]
Hi, most of the time, in our routine is necessary to generate a DVD media with all evidence found out and bookmarked (after they match specific criteria) in analyzed case.
In this way, it would be a great feature the application to provide a kind of 'viewer' showing the size of the items bookmarked (maybe a tooltip on hover the button of items bookmarked at the right top of "IPED-SearchApp.exe").
With this information, the user will be able to truncate or to discharge the items that exceed the size of DVD media.

**Comment by lfcnassif** [comment]
Doesn't it work to select all bookmarks (using shift or ctrl), applying duplicate filter (iped does not export duplicate files twice) and checking total listed items size in 'Size' table header?

About future index size, that is impossible to predict, but usually is about 20%-30% of total indexed text size (not item size). Images and videos result in much less text than documents.

**Comment by danpos** [comment]
> Doesn't it work to select all bookmarks (using shift or ctrl), applying duplicate filter (iped does not export duplicate files twice) and checking total listed items size in 'Size' table header?

No, it doesn't.

**Comment by lfcnassif** [comment, marked as answer]
and why? You can just select your bookmarks and look at "size" column table header:
![image](https://user-images.githubusercontent.com/7276994/86516568-1334fc00-bdf8-11ea-8fdd-1e96b359bf43.png)

**Comment by danpos** [comment]
Ah, I see it now! Thank you!

**Comment by lfcnassif** [comment]
You're welcome
